service/scaladsl/kafka/client/src/main/scala/com/lightbend/lagom/internal/scaladsl/broker/kafka/KafkaTopicFactory.scala

```diff
@@ -26,17 +25,7 @@ private[lagom] class KafkaTopicFactory(
     config: Config
 )(implicit materializer: Materializer, executionContext: ExecutionContext)
     extends TopicFactory {
-  @deprecated("Use constructor that accepts a Config", "2.0.0")
-  def this(serviceInfo: ServiceInfo, system: ActorSystem, serviceLocator: ServiceLocator)(
-      implicit materializer: Materializer,
-      executionContext: ExecutionContext
-  ) = {
-    this(serviceInfo, system, serviceLocator, system.settings.config)
-  }
```

Comment:
can this be removed already?

If I'm reading the deprecation warning correctly, it was just deprecated in `master` (but wrongfully labeled as `deprecated since` "2.0.0"). This should be labeled as `"1.7.0"` and kept until `1.8.0`.